iped-parsers/iped-parsers-impl/src/test/java/dpf/mg/udi/gpinf/vcardparser/VCardParserTest.java

```diff
@@ -197,12 +196,13 @@ public void testVCardCompleteHTMLToString() throws IOException, SAXException, Ti
         Metadata metadata = new Metadata();
         ContentHandler handler = new BodyContentHandler();
         ClassLoader classLoader = getClass().getClassLoader();
-        File file = new File(
-                classLoader.getResource("test-files/test_contactsCompleteWithoutThumbHTMLToString.vcf").getFile());
-        PrintWriter out = new PrintWriter(file);
-        ParseContext context = new ParseContext();
-        parser.getSupportedTypes(context);
         try(InputStream stream = getStream("test-files/test_contactsCompleteWithoutThumbHTMLToString.vcf")){
+	        File file = new File(
```

Comment:
@streeg this method has many errors: stream closed twice, stream.toString() and seems it is trying to overwrite the testing file.

Reply:
Just learned that try block closes InputStream automatically. Didn't knew about the java.lang.AutoCloseable feature. Sorry, I already removed them. About VCardParserTest, my mistake with the stream, I might have mispelled by accident.

Reply:
Thank you!